.env

```diff
@@ -4,4 +4,3 @@ OAUTH_PROVIDER_URL=https://isso.nypl.org/oauth/token
 OAUTH_CLIENT_ID=askTeammate
 OAUTH_CLIENT_SECRET=askTeammate
 IPSTACK_KEY=askTeammate
-NEXT_PUBLIC_USE_AXE=false
```

Comment:
was there an issue with using axe?

Reply:
No. It's just not a tool that's used. The jest-axe package is equivalent and runs on every test run.